docs/book/getting_started/introduction.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Start here with the ZenML Docs.
+---
+
+# Introduction
+
+Are you an ML engineer or data scientist shipping models to production and jumbling a plethora of tools?
+Do you struggle with versioning data, code, and models in your projects?
+Have you had trouble replicating production pipelines and monitoring models in production?
+
+If you answered yes to any, **ZenML** is here to help with all that, and more. 
+![Zenml](../assets/oss-header.svg)
+
+**ZenML** is an extensible, open-source MLOps framework for creating portable, production-ready **MLOps pipelines**. 
+It's built for data scientists, ML Engineers, and MLOps Developers to collaborate as they develop to production.
+**ZenML** has simple, flexible syntax, is **cloud-** and 
+**tool-agnostic**, and has interfaces/abstractions that are catered towards ML workflows.
+
+
+<!-- At its core, ZenML pipelines execute ML-specific workflows from sourcing
+data to splitting, preprocessing, training, all the way to serving and monitoring 
+ML models in production. There are many built-in features to support
+common ML development tasks. ZenML is not here to replace the great tools that
+solve these individual problems. Rather, it offers an **extensible framework** and a standard abstraction to write and build your workflows. -->
+
+## Why ZenML
+
+ZenML pipelines are designed to be written early in the development lifecycle. 
+As a data scientist, you can explore and tailor your pipelines as you develop toward production.
+Once you're satisfied, you can switch from local to cloud deployment.
+<!-- Read more about why we started building ZenML on our [blog](https://blog.zenml.io/why-zenml/). -->
+
+Here's what you get by using ZenML -
+
++ Extensibility - Build a framework to suit your specific needs.
++ Reproducibility - Replicate training and inference workflows.
++ Clarity - Represent steps of your pipeline in code.
++ Batteries-included integrations - Bring all your favorite tools together.
++ Easily switch between local and cloud stacks.
++ Painless deployment and infrastructure configuration.
+
+Eager to get started? We recommend the following to start using **ZenML** right away!
+
+## **Pipelining** - Take your first ZenML Steps
+
+If you are new to ZenML, you might want to
+get acquainted with the [core concepts](core-concepts.md) first 
+to understand what makes ZenML so special. Next, jump right 
+into our [developer guide](../developer-guide/installation.md) that will 
+take you from zero to hero in no time and explain the necessary knowledge of 
+**how to use ZenML**.
+
+## **Stacking** - Customize your Stack
```

Comment:
This might rephrasing: A stack doesnt mean much to externals. Maybe write "Stacking - Connect tools of your choice"